presentation/src/main/java/dk/nodes/template/presentation/ui/splash/SplashViewModel.kt

```diff
@@ -15,10 +15,12 @@ import javax.inject.Inject
 
 class SplashViewModel @Inject constructor(
     private val nStackPresenter: NStackPresenter
-) : BaseViewModel<SplashViewState>() {
-
-    override val initState: SplashViewState =
-        SplashViewState(doneLoading = false, nstackUpdateAvailable = null)
+) : BaseViewModel<SplashViewState>(
```

Comment:
Not sure this looks better than overriding the val.

Reply:
It's not only because of looks, this makes the initial state be emitted on start
Before we would only get the value on the LiveData after some change has been made on the state.